v2/paths/recognition/ClusterSetCompute.js

```diff
@@ -1,17 +1,17 @@
 module.exports = {
   post: {
-    operationId: 'face-recognition/cluster-set-compute',
+    operationId: 'recognition/cluster-set-compute',
```

Comment:
You can remove all `Cluster[Set]` files as they're no longer part of the new API.

Reply:
Removed all the Cluster related docs